### ISSUE TITLE
`requestIdleCallback` is not correctly reported, it's not supported in all ios safari versions.

### ISSUE THREAD

**Issue by latel** [amilajack/eslint-plugin-compat#670]
.browserslistrc
``` 
iOS >= 11.3
android >= 63
not IE > 0
not and_qq > 0
not and_uc > 0
not kaios > 0
not op_mob > 0
not samsung > 0
```

``` bash
pnpm exec eslint --stdin --stdin-filename temp.ts <<'EOF'
requestIdleCallback(() => {});
EOF

(node:30160) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
```
